Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_3.h

```diff
@@ -220,47 +220,21 @@
 #define EXP2_07_PIN                        P1_31
 #define EXP2_08_PIN                        -1
 
-#if HAS_WIRED_LCD
```

Comment:
SKR 1.3 when using CR10_STOCKDISPLAY now sets a KILL_PIN, which it previously did not. I don't recall wither this display supports a reset button or not.

Old:
`#define KILL_PIN -1`

New:
`#define KILL_PIN LCD1_04_PIN // Maps to P1_30`

Reply:
I saw this same behavior with CR10_STOCKDISPLAY on other boards as well.

Reply:
no reset button visible https://marlinfw.org/docs/hardware/controllers.html#cr10_stockdisplay

Reply:
> But things like btt tft35 have a reset button and emulate a cr10_stockdisplay
> So its probably good to define a kill pin.

Ahh, yes. I forgot about that. They have a reset button & it works!

<img width="688" alt="image" src="https://github.com/MarlinFirmware/Marlin/assets/13375512/cd076738-c362-405c-b508-9b0148419efc">

Reply:
Also the [CR10_schematic](https://github.com/RudolphRiedel/CR-10_wiring/blob/master/Ender3_schematic.PDFl)

shows the following, win a reset pin defined. 

<img alt="Screen Shot 2019-08-19 at 9 41 40 PM" width="353" src="https://user-images.githubusercontent.com/13375512/63318128-34692880-c2ca-11e9-9fb3-e8087ae3290f.png">

Reply:
With the new adapters and individual display definitions we'll get the correct result more often. For boards that have a RESET line on the EXP port the display reset button will get mapped to it, and if they don't then it will just be mapped to -1.